com.unity.render-pipelines.high-definition/Documentation~/whats-new-11.md

```diff
@@ -20,6 +20,14 @@ Cubemap fields now accept both [RenderTextures](https://docs.unity3d.com/Manual/
 
 For more information, see the [HDRI Sky](Override-HDRI-Sky.md) and [Physically Based Sky](Override-Physically-Based-Sky) sections of the documentation.
 
+### Density Volume Improvements
+
+Density Volumes masks has been reworked to allow 3D RenderTextures to be used as masks. 3D Mask textures now uses all four RGBA channel which allows volumetric fog to have different colors and density based on the 3D Texture.
+
+The size limit of 32x32x32 for the mask textures has also been replaced by a setting in the HDRP asset called "Max Density Volume Resolution", under the lighting > Volumetrics tab. The upper limit for mask textures is now 256x256x256, an info box below the field will tell you how much memory is allocated to store those textures.
```

Comment:
The size limit of 32x32x32 for the mask textures has also been replaced by a setting in the HDRP asset called "Max Density Volume Resolution", under the **Lighting > Volumetrics** section. The upper limit for mask textures is now 256x256x256, an info box below the field tells you how much memory is allocated to store these textures.